KerbalStuff/blueprints/mods.py

```diff
@@ -31,6 +32,10 @@
 
 mods = Blueprint('mods', __name__, template_folder='../../templates/mods')
 
+SOURCE_REPOSITORY_URL_PATTERN = re.compile(
+    r'^https://git(hub|lab).com/(?P<repo_short>[^/]+/[^/]+)/?$'
```

Comment:
I think I've seen several mods include suffixes in this field, such as:

- https://github.com/KSP-CKAN/CKAN/releases

Can we remove the `$` to allow the regex to match less than the full string?

Reply:
Fair enough, these URLs also have a right to be shortened.